webrender/src/renderer.rs

```diff
@@ -5340,6 +5377,7 @@ impl<B: hal::Backend> Renderer<B> {
                                         frame_id,
                                         &mut results.stats,
                                         false,
+                                        false,
```

Comment:
would be nicer to have this as an enum, so that we can see the meaning at the call site

Reply:
Added a `TargetKind` enum for this purpose: https://github.com/szeged/webrender/pull/337/files#diff-f5062b694b9fe53fc1757ed483d577d9R342